jaxtyping/_ipython_extension.py

```diff
@@ -0,0 +1,28 @@
+try:
+    from IPython.core.magic import line_magic, Magics, magics_class
+except ImportError:
+    pass
+
+from ._import_hook import _JaxtypingTransformer
+
+
+@magics_class
```

Comment:
Alright, I think the decorators are going to fail if IPython is not imported. Going to fix it now...

Reply:
Now should be alright.